### PR TITLE
list additional issues

### LINKED ISSUE #4589: Modules and Plugins with Known Issues Doc Update
Re: [Modules and Plugins with Known Issues](https://pantheon.io/docs/modules-plugins-known-issues/)

Priority: Low

## Issue Description:
A customer searched for a plugin they wanted to use on the Known Issues page, did not find it, and assumed therefore that it would work, even though the plugin's documentation clearly stated that it was only supported on Apache web servers.

## Suggested Resolution
It is not possible to identify and list every plugin that will not work on the platform. It might be helpful to have an overview at the top of this page listing things to look out for when evaluating modules and plugins that do not appear on the list.

Some examples:
- Requires Apache
- Uses .htaccess files
- Requires changing Nginx configuration
- Requires Postgresql or other non-Mysql database

A link to the information at the bottom of the page somewhere near the top with an appropriate call-out might also be helpful.

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #4589

## Effect
PR includes the following changes:
- Note common problems with plugins and modules.

## Remaining Work
- [x] Technical review
- [x] Copy Review

## Post Launch
To be completed by the docs team upon merge: 
- [ ] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

**Comment by greg-1-anderson** [comment]
LGTM, but it might be a good idea to get a different technical reviewer for this.

**Comment by alexfornuto** [comment]
Who do you have in mind @greg-1-anderson ?

**Comment by greg-1-anderson** [comment]
Maybe @stevector or someone else from #dev-rel?

File: source/_docs/modules-plugins-known-issues.md [hunk 1]
```diff
@@ -794,3 +794,12 @@ Due to the cloud-based infrastructure of the Pantheon platform, outbound request
 **Issue**: Extensions that require the use of the `/tmp` directory are not supported. With multiple application containers, as exists on Live environments, it's assumed the `/tmp` directory will be on the same application container. However, as we run a distributed application container matrix, the `/tmp` directory is not shared.
 
 **Solution**: For more details, see [Temporary File Management](/docs/tmp/).
+
+## Other Issues
+
+Plugins and modules will not work on Pantheon if they:
+
+ - Require Apache
+ - Use `.htaccess` files
```
Comment (stevector) [inline source/_docs/modules-plugins-known-issues.md]:
```suggestion
 - Require customized `.htaccess` files
```

**Comment by stevector** [comment]
My first reaction upon reading the change was "this seems like an overly verbose way of saying 'all Pantheon sites get the same nginx config'" but it's probably best to spell it out in a list like this for clarity sake.

**Pull request merged**